File: C Modules/Assignments/Assignment no 7/Question-1/Type_3/Assignment no 2/question-4.c
#include<stdio.h>
int students_marks()
{
   int marks;
   int *ptr1=&marks;
   printf("Enter the marks:");
   scanf("%d",ptr1);
   if (*ptr1>75)
   {
      return 0;
   }
   else if (*ptr1>65)
   {
   return 1;
   }
   else if (*ptr1>55)
   {
    return 2;
   }
   else if (*ptr1>=40)
   {
    return 3;
   }
   else if (*ptr1<=40)
   {
    return 4;
   }
}
int main()
{
   int result=students_marks();
   if (result==0)
   {
    printf("Distinction");
   }
   else if (result==1)
   {
    printf("First Class");
   }
   else if (result==2)
   {
    printf("Second Class");
   }
   else if (result==3)
   {
    printf("Pass Class");
   }
   else if (result==4)
   {
    printf("Fail");
   }
   return 0;
}

File: Java Module/Assignments in c to java/Assignment-02/question_4.java
public class question_4 {
public static void main(String[] args) {
       int marks=45;
//    printf("Enter the marks:");
//    scanf("%d",&marks);
   if (marks>75)
   {
    System.out.printf("Distinction",marks);
    /* code */
   }
   else if (marks>65)
   {
    System.out.printf("First Class",marks);
    /* code */
   }
   else if (marks>55)
   {
    System.out.printf("Second Class",marks);
    /* code */
   }
   else if (marks>=40)
   {
    System.out.printf("Pass Class",marks);
    /* code */
   }
   else if (marks<=40)
   {
    System.out.printf("Fail",marks);
    /* code */
   }
}
   
}